glide.yaml

```diff
@@ -2,7 +2,7 @@ package: github.com/thingful/testharness
 import:
 - package: github.com/thingful/httpmock
 - package: github.com/thingful/thingfulx
-  version: 0.1.9
+  version: 0.2.0
```

Comment:
glide does have concept of patch ranges, so you could change this to:

```
version: ~0.2.0
```

which is the same as: `>= 0.2.0` && `< 0.3.0`